### PR TITLE
Adetoyebamise patch 1

### PR DESCRIPTION
**Pull request by Adetoyebamise**
Adetoyebamise Javascript

### CODE REVIEW

**Comment by dro-1** [comment]
Wrong Output

File: adetoyebamise.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const requirements ={
+   
+      firstName: "Adetoye",
+      lastName: "Babalola",
+      HNGi7_ID: "HNG-05584",
+      language: "Javascript",
+      email: "adetoyebamise@gmail.com"
+    }
+    const statement = `Hello World, this is ${requirements.firstName} ${requirements.lastName} with HNGi7 ID ${requirements.HNGi7_ID} and email ${requirements.email} using ${requirements.language} for stage2 task`;
```
Comment (simply-alliv) [inline adetoyebamise.js]: Ensure that the words "stage" and "2" are separated.

File: scripts/Adetoyebamise.js.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const requirements ={
```
Comment (simply-alliv) [inline scripts/Adetoyebamise.js.txt]: You do not need to have two files of the same thing. Please delete this extra file.

Comment (Rubylena) [reply scripts/Adetoyebamise.js.txt]: delete this file ade. thanks.
am passing

**Comment by Adetoyebamise** [comment]
Done

**Comment by Adetoyebamise** [comment]
Help review again @Rubylena

**Comment by maroafenogho** [comment]
I think this is good to go. @dro-1 please confirm

**Comment by simply-alliv** [comment]
The file needs to be moved into the scripts folder first

**Comment by simply-alliv** [comment]
The file needs to be moved to the scripts folder first

**Pull request merged**